src/main/java/uk/nhs/digital/nhsconnect/nhais/configuration/AppConfiguration.java

```diff
@@ -21,7 +21,10 @@
 
     @Bean
     public AmazonS3 getS3Client() {
-        return AmazonS3ClientBuilder.standard()
-            .build();
+        if(trustStoreUrl != null && trustStoreUrl.startsWith("s3")) {
```

Comment:
Why this change? Client initialization fails if AWS is not available or is this just an optimization step?

Reply:
App doesn't start if there are no AWS environment variables on the PATH. In this case, it is the region that is required.